pyston/pyston_lite/Makefile

```diff
@@ -51,6 +51,7 @@ BENCH:=../macrobenchmarks/benchmarks/bm_flaskblogging/run_benchmark.py --legacy
 # note: the testsuite doesn't pass for my Ubuntu system cpython, but it does pass
 # if I reconfigure this makefile to use a manually-built cpython
 testsuite: env
+	PYSTON=1 ./env/bin/python -c 'import test.support; test.support.check_impl_detail = lambda **kw: False; import test.test_code; test.test_code.test_main()'
```

Comment:
It can't fix test_code for me not sure how I have to generate it.

Reply:
I had to do `sudo apt-get install libpython3.8-testsuite` since I think Ubuntu strips out the tests from their package. If you test this against a manually-built cpython I think it just works.

Reply:
a thanks that fixed it